django_snowflake/features.py

```diff
@@ -23,8 +22,8 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     # This feature is specific to the Django fork used for testing.
     supports_indexes = False
     supports_index_column_ordering = False
-    # Not yet implemented in this backend.
-    supports_json_field = False
+    # TODO: Not yet implemented in this backend.
+    supports_json_field_contains = False
```

Comment:
Some other databases implement `JSON_CONTAINS()` (`@>` and `<@` on PostgreSQL). Is it unsupported by Snowflake or did I miss an equivalent?

Reply:
You can treat JSON columns as regular `VARIANT` columns, like:
```sql
select *
from (
  select parse_json(column1) as user
  from (
    select * from values
    ('{"name": "Luke", "age": 25}'),
    ('{"name": "Mark", "age": 35}')
  )
) where user ilike '%luke%'
;
```
We don't have any special JSON variants of this as far as I know

Reply:
I'm not sure we can (easily?) use regular expressions since the example code you gave will also match JSON keys. I'll call this unsupported, at least for now.